src/main/java/seedu/application/logic/commands/EditCommand.java

```diff
@@ -30,21 +42,24 @@ public class EditCommand extends Command {
         + "[" + PREFIX_COMPANY + "COMPANY] "
         + "[" + PREFIX_DEADLINE + "DEADLINE] "
         + "[" + PREFIX_STATUS + "STATUS] "
+        + "[" + PREFIX_JOBTYPE + "JOB_TYPE] "
         + "[" + PREFIX_INDUSTRY + "INDUSTRY] "
         + "Example: " + COMMAND_WORD + " 1 "
         + PREFIX_ROLE + "Software Engineer "
-        + PREFIX_COMPANY + "Google "
```

Comment:
should retain the whitespace after the string if not the message is going to be quite weird